cedar-policy-core/src/ast/name.rs

```diff
@@ -89,6 +89,24 @@ impl Name {
     pub fn namespace(&self) -> String {
         self.path.iter().join("::")
     }
+
+    pub fn prefix_namespace(&self, namespace: Option<Name>) -> Name {
```

Comment:
Can we get a doc comment for this method?  Even better if it includes a couple examples, like the doc comment for the function above